tests/e2e/workloads/couchbase/test_couchbase.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import E2ETest, workloads, ignore_leftovers
+from ocs_ci.ocs.couchbase import CouchBase
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def couchbase(request):
+
+    couchbase = CouchBase()
+
+    def teardown():
+        couchbase.teardown()
+    request.addfinalizer(teardown)
+    return couchbase
+
+
+@workloads
+@ignore_leftovers
```

Comment:
any reason for ignoring leftovers?

Reply:
need to remove @ignore_leftovers

Reply:
ignore_leftovers came with old test_couchbase.py. AFAIK, there should be no leftovers. Will check and update

Reply:
no leftovers, removed @ignore_leftovers